test/authenticator.spec.ts

```diff
@@ -286,15 +264,13 @@ describe('Decentraland Crypto', function () {
       const result = await Authenticator.validateSignature(
         'QmbGrShBQs4XiuoTNX6znAvXNdqtub8DtXyaxdSTZbHLCu',
         chain,
-        new HttpProvider(
-          'https://mainnet.infura.io/v3/640777fe168f4b0091c93726b4f0463a'
-        )
+        new HTTPProvider('https://rpc.decentraland.org/mainnet')
```

Comment:
Can we use https://rpc.decentraland.org/mainnet?project=dcl-crypto-test so we can discriminate the usage?